### DISCUSSION TITLE
Welcome to MUSE Discussions!

### DISCUSSION THREAD

**Discussion by zhangh3** [zhangh3/MUSE#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lew1se** [comment]
初步了解如何使用muse，但是对详细使用还有很多不懂的地方，对于添加其他约束有一些打算
